lib/mindwendel_web/live/idea_live/form_component.html.leex

```diff
@@ -27,8 +28,8 @@
   </div>
 
   <%= hidden_input f, :brainstorming_id %>
-
-  <br /><br />
-  <%= live_patch gettext("Close"), to: @return_to, class: "btn btn-secondary form-cancel" %>
-  <%= submit gettext("Save"), phx_disable_with: gettext("Saving..."), class: "btn btn-primary" %>
 </form>
+<br />
+
+<%= live_patch gettext("Close"), to: @return_to, class: "btn btn-secondary form-cancel me-2"  %>
+<%= submit gettext("Save"), phx_disable_with: gettext("Saving..."), class: "btn btn-primary", disabled: !@changeset.valid? %>
```

Comment:
I noticed that the Saving an idea does not work. Probably, it has something to do with moving the submit button out of the form.

Reply:
can you try again? I have use a different approach, it is possible to ignore some elements from the live patch...

Interesting: https://hexdocs.pm/phoenix_live_view/dom-patching.html